### PR TITLE
Move glTF export option under scene menu

### PR DESCRIPTION
**Pull request by techiepriyansh**
Documentation updates for godotengine/godot#59393.

Results:
image
image

### CODE REVIEW

**Comment by Calinou** [comment]
Note: I converted this PR to draft to make sure this isn't accidentally merged before https://github.com/godotengine/godot/pull/59393 is also merged.

**Comment by techiepriyansh** [comment]
godotengine/godot#59393 has been merged now.

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

File: tutorials/3d/using_gridmaps.rst [hunk 1]
```diff
@@ -65,7 +65,7 @@ library. Materials set on the node will be ignored.
 Exporting the MeshLibrary
 -------------------------
 
-To export the library, click on Scene -> Convert To.. -> MeshLibrary.., and save it
+To export the library, click on **Scene -> Export As... -> MeshLibrary...**, and save it
```
Comment (Calinou) [inline tutorials/3d/using_gridmaps.rst:68]:
```suggestion
To export the library, click on **Scene > Export As... > MeshLibrary...**, and save it
```

**Pull request merged**